Header header

string[]        N_ID
string[]        A_ID
int32           NUM_A
float64[]       A_POS
string          T_ID
float64[]       D
